test/evergreen/setup_spawn_host.sh

```diff
@@ -23,6 +23,21 @@ if [[ -n $WT_ARCHIVE ]]; then
     tar --wildcards -xzf $WT_ARCHIVE
 fi
 
+# Setup the gdb environment if there are core dumps present in the artefacts.
+GDB_CORE_DUMP=$(ls ${HOME}/wiredtiger/cmake_build/*.core | head -n 1 2>/dev/null)
```

Comment:
Sometimes the core file can be in test/format, or test/suite, etc.

Reply:
fixed! [2e15df9](https://github.com/wiredtiger/wiredtiger/pull/9556/commits/2e15df95daab3de0e4361859368b98c7bf7f700c)